src/tools/kdb/mountpoint-mount.c

```diff
@@ -45,51 +43,191 @@ void addMountSpec (KeySet * spec)
-	ksAppendKey (spec, keyNew (COMMAND_SPEC_KEY (COMMAND_NAME) "/nullterm", KEY_META, "description", "Use binary 0 termination.",
-				   KEY_META, "opt", "0", KEY_META, "opt/long", "null", KEY_META, "opt/arg", "none", KEY_END));
-
-	ksAppendKey (spec, keyNew (COMMAND_SPEC_KEY (COMMAND_NAME) "/supfirst", KEY_META, "description", "Suppress the first column.",
-				   KEY_META, "opt", "1", KEY_META, "opt/long", "first", KEY_META, "opt/arg", "none", KEY_END));
-	ksAppendKey (spec, keyNew (COMMAND_SPEC_KEY (COMMAND_NAME) "/supsecond", KEY_META, "description", "Suppress the second column.",
-				   KEY_META, "opt", "2", KEY_META, "opt/long", "second", KEY_META, "opt/arg", "none", KEY_END));
-	ksAppendKey (spec, keyNew (COMMAND_SPEC_KEY (COMMAND_NAME) "/supthird", KEY_META, "description", "Suppress the third column.",
-				   KEY_META, "opt", "3", KEY_META, "opt/long", "third", KEY_META, "opt/arg", "none", KEY_END));
 
-	ADD_BASIC_OPTIONS (spec, COMMAND_SPEC_KEY (COMMAND_NAME))
+	ADD_BASIC_OPTIONS (spec, COMMAND_SPEC_KEY (COMMAND_NAME));
 }
 
 int execMount (KeySet * options, Key * errorKey)
 {
-	int ret = 0;
-	bool verbose = false;
-	Key * tmp = GET_OPTION_KEY (options, "verbose");
-	if (tmp != NULL)
+	/* Options that can be set via command line arguments */
+	/* 1. Standard options */
+	/* TODO: Discuss the evaluation of global option (not specific to command) */
+	bool optDebug = false;
+	bool optVerbose = false;
+	bool optVersion = false;
+	bool optSuppressNewline = false;
+	/* TODO: Implement option */
+	const char * optProfile = NULL;
+	/* TODO: Implement option */
+	const char * optColor = NULL;
+
+	/* 2. Command-specific options */
+	bool optForce = false;
+	bool optQuiet = false;
+	bool optInteractive = false;
+	bool optWithRecommends = false;
+	/* TODO: Decide where to put code for evaluation of option (currently also in cmerge.c) */
+	const char * optStrategy = NULL;
+	/* TODO: Implement option */
+	const char * optResolver = NULL;
+	int mergeStrategy = MERGE_STRATEGY_ABORT;
+
+	/* 3. Command arguments ("[path mountpoint] [plugin [config] [..]]" */
+	/* current simplified parameters: "kdb mount [options] [path mountpoint] [plugin]" */
+	const char * argPath = NULL;
+	const char * argMountpoint = NULL;
+
+	const Key * tmp;
+	if ((tmp = GET_OPTION_KEY (options, "debug")))
+		elektraKeyToBoolean (tmp, &optDebug);
+	if ((tmp = GET_OPTION_KEY (options, "verbose")))
+		elektraKeyToBoolean (tmp, &optVerbose);
+	if ((tmp = GET_OPTION_KEY (options, "version")))
+		elektraKeyToBoolean (tmp, &optVersion);
+	if ((tmp = GET_OPTION_KEY (options, "nonewline")))
+		elektraKeyToBoolean (tmp, &optSuppressNewline);
+
+	/* command-specific options */
+	if ((tmp = GET_OPTION_KEY (options, "force")))
+		elektraKeyToBoolean (tmp, &optForce);
+	if ((tmp = GET_OPTION_KEY (options, "quiet")))
+		elektraKeyToBoolean (tmp, &optQuiet);
+	if ((tmp = GET_OPTION_KEY (options, "interactive")))
```

Comment:
AFAIK we planned to remove the interactive mode, since nobody is using it anyway